src/http-servers/html-server.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+const index = fs.readFileSync('index.html');
```

Comment:
remove it if you will use readStream. again, unused variable. are you sure that your IDE runs ESLint?

Reply:
I assumed that task with html sever is split to 2 subtask:
'Set Content-Type header to html.
(1)Read (readFileSync) an index.html file with fs module, replace message with a real message text.
Send the response.
(2)Change readFileSync to be a readable stream and pipe it to response stream.'
so I left index var because it is used in the commented version of the code.

I will delete this index var and all that is commented because it is not used. 👍

Reply:
ok